pybleau/app/ui/dataframe_plot_manager_view.py

```diff
@@ -5,17 +5,22 @@
 
 from enable.component_editor import ComponentEditor
 from pyface.api import warning
-from traits.api import Any, Bool, Button, Enum, HasStrictTraits, Instance, \
-    Int, List, Str
+from traits.api import Any, Bool, Button, Enum, Instance, \
+    Int, List
 from traitsui.api import EnumEditor, HGroup, Item, Label, ModelView, \
-    OKCancelButtons, Spring, TableEditor, VGroup, View, VSplit, ObjectColumn
+    Spring, TableEditor, VGroup, View, VSplit, ObjectColumn
 from traitsui.extras.checkbox_column import CheckboxColumn
+from traitsui.menu import CancelButton, OKButton
 
 from pybleau.app.model.dataframe_plot_manager import CONTAINER_IDX_REMOVAL, \
     DataFramePlotManager
 from pybleau.app.model.multi_canvas_manager import CONTAINER_TRAIT_NAME
+from pybleau.app.model.tests.test_plot_template_manager import \
+    FakePlotTemplateInteractor
```

Comment:
Please move this to the `if name = main` section since only used there.

Reply:
Done.